scripts/cube_fx.lua

```diff
@@ -290,11 +300,59 @@ local function cube_victory(size, results)
   end
 end
 
+-- Map to determine if some machine is using the cube
+-- in an "efficient" manner. 
+local cube_machine_condition_handlers = {
```

Comment:
I'd prefer if the logic was just inlined into the `track_victory_statistics` function, like
```
if crafter_entity_types[entity_type] then
  // crafter utilization code
elseif burner_types[entity_type] then
  // burner utilization code
end
```
unless you have good reason to think your way is better for performance. I suspect function call overhead would be more than some map checks.

Reply:
I did it for performance reasons. If you use a large `if ellse` chain then every `if` needs to be executed until the one we need is found. Whereas with my approach if can find the function directly.

That said, this list likely will be less than 10 long, so I'm not sure if it's really that much worse performance-wise.